src/main/java/io/anserini/document/Status.java

```diff
@@ -0,0 +1,213 @@
+package io.anserini.document;
+
+import java.util.List;
+import java.util.Optional;
+import java.util.OptionalDouble;
+import java.util.OptionalLong;
+
+import com.fasterxml.jackson.annotation.*;
+import com.fasterxml.jackson.databind.*;
+
+/**
+ * A Twitter document (status) class used in Jackson JSON parser
+ */
+public class Status {
+
+  // Required fields
+  protected String created_at;
+  protected String id_str;
+  protected String text;
+  protected User user;
+
+  protected long retweet_count;
```

Comment:
Is `retweet_count` a required field?

Reply:
Nope

Reply:
Then I think we should make it as OptionalLong, no?

Reply:
It doesn't have to be `OptionalLong`. When the value is missing during deserialization, we will get `null` if we try to access it.

If we make this `OptionalLong`, then we should make all optional fields to have `Optional` type. Do you want this?

Reply:
I just saw you made other fields below as Optional fields. 
Shouldn't we be consistent here?

Reply:
Right, I'll make this change.